### PR TITLE
fix(react-amp): don't wrap img with <picture> in amp mode

### PR DESCRIPTION
**Pull request by ca136**

### CODE REVIEW

File: packages/react/src/blocks/Image.tsx [hunk 1]
```diff
@@ -99,65 +99,71 @@ class ImageComponent extends React.Component<any> {
...
+                      this.props.image.split('?')[0]) ||
+                    undefined
+                  : undefined
+              }
+              // height={
+              //   this.props.height || (aspectRatio ? Math.round(aspectRatio * 1000) : undefined)
+              // }
+              // width={
+              //   this.props.width || (aspectRatio ? Math.round(1000 / aspectRatio) : undefined)
+              // }
```
Comment (steve8708) [inline packages/react/src/blocks/Image.tsx]: comments here can be removed

File: packages/react/src/blocks/Image.tsx [hunk 2]
```diff
@@ -99,65 +99,71 @@ class ImageComponent extends React.Component<any> {
...
-                    }}
-                    className="builder-image"
-                    src={this.props.image}
-                    // TODO: memoize on image on client
-                    srcSet={srcset}
-                    sizes={this.props.sizes}
-                  />
-                )}
-              </picture>
+              { amp ?
```
Comment (steve8708) [inline packages/react/src/blocks/Image.tsx]: this is backwards (amp should not show picture)

File: packages/react/src/blocks/Image.tsx [hunk 3]
```diff
@@ -99,65 +99,71 @@ class ImageComponent extends React.Component<any> {
         {value => {
           const amp = value.ampMode
           const Tag: 'img' = amp ? ('amp-img' as any) : 'img'
+
+          const ImageContents = (!lazy || this.state.load) && (
```
Comment (steve8708) [inline packages/react/src/blocks/Image.tsx]: this should be lowercase `imageContents`

File: packages/react/src/blocks/Image.tsx [hunk 4]
```diff
@@ -99,65 +99,71 @@ class ImageComponent extends React.Component<any> {
...
+                  position: 'absolute',
+                  height: '100%',
+                  width: '100%',
+                  left: 0,
+                  top: 0
+                }),
+                ...(amp && {
+                  ['& img']: {
+                    objectFit: this.props.backgroundSize,
+                    OObjectPosition: this.props.backgroundPosition
```
Comment (steve8708) [inline packages/react/src/blocks/Image.tsx]: oh this is actually a typo from the original code - should be `objectPosition: ...`

File: packages/react/src/blocks/Image.tsx [hunk 5]
```diff
@@ -99,65 +99,64 @@ class ImageComponent extends React.Component<any> {
...
+                ...(amp && {
+                  ['& img']: {
+                    objectFit: this.props.backgroundSize,
+                    ObjectPosition: this.props.backgroundPosition
+                  }
+                })
+              }}
+              className="builder-image"
+              src={this.props.image}
+              // TODO: memoize on image on client
```
Comment (ca136) [inline packages/react/src/blocks/Image.tsx]: @steve8708 you want this comment removed as well?

**Comment by ca136** [comment]
@steve8708 updated, didn't add the unit test yet, will follow-up with a new PR for that and fix the tests for Reach SDK (breaking due to:
```
Cannot find module 'lib/on-change.js' from 'builder-page.component.js'
```

File: packages/react/src/blocks/Image.tsx (hunk 5 above)
Comment (steve8708) [reply packages/react/src/blocks/Image.tsx]: this one we can keep actually, would be good to add at some point but not urgent

**Comment by steve8708** [comment]
ah yes I need to remove that file from being git ignored, will add to todo list :)

PR looks great!

**Pull request merged**